app/src/main/java/io/appium/uiautomator2/handler/Swipe.java

```diff
@@ -1,51 +1,91 @@
 package io.appium.uiautomator2.handler;
 
+import android.support.test.uiautomator.UiDevice;
+import android.support.test.uiautomator.UiObjectNotFoundException;
+
 import org.json.JSONException;
 import org.json.JSONObject;
 
+import io.appium.uiautomator2.common.exceptions.InvalidCoordinatesException;
 import io.appium.uiautomator2.handler.request.SafeRequestHandler;
 import io.appium.uiautomator2.http.AppiumResponse;
 import io.appium.uiautomator2.http.IHttpRequest;
+import io.appium.uiautomator2.model.AndroidElement;
+import io.appium.uiautomator2.model.KnownElements;
 import io.appium.uiautomator2.server.WDStatus;
 import io.appium.uiautomator2.utils.Device;
 import io.appium.uiautomator2.utils.Logger;
+import io.appium.uiautomator2.utils.Point;
+import io.appium.uiautomator2.utils.PositionHelper;
 
 public class Swipe extends SafeRequestHandler {
+    UiDevice device;
 
     public Swipe(String mappedUri) {
         super(mappedUri);
     }
 
     @Override
     public AppiumResponse safeHandle(IHttpRequest request) {
+        Point absStartPos, absEndPos;
```

Comment:
Minor: declare the variable where its being first initialized, by this we narrow the scope of the variable. Same goes with swipeArgs as well.